src/main/cpp/mechanisms/climber/ClimberReadyState.cpp

```diff
@@ -0,0 +1,67 @@
+//====================================================================================================================================================
+// Copyright 2024 Lake Orion Robotics FIRST Team 302
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+// OR OTHER DEALINGS IN THE SOFTWARE.
+//====================================================================================================================================================
+// This file was automatically generated by the Team 302 code generator version 1.2.3.6
+// Generated on Sunday, February 18, 2024 12:51:47 PM
+
+// C++ Includes
+#include <string>
+
+// FRC includes
+
+// Team 302 includes
+#include "mechanisms/climber/ClimberReadyState.h"
+#include "teleopcontrol/TeleopControl.h"
+#include "teleopcontrol/TeleopControlFunctions.h"
+#include "utils/logging/Logger.h"
+using namespace ClimberConstants;
+
+// Third Party Includes
+
+using namespace std;
+
+/// @class ExampleForwardState
+/// @brief information about the control (open loop, closed loop position, closed loop velocity, etc.) for a mechanism state
+ClimberReadyState::ClimberReadyState(std::string stateName,
+									 int stateId,
+									 CANClimber *mech) : State(stateName, stateId), m_mechanism(mech)
+{
+}
+
+void ClimberReadyState::Init()
+{
+}
+
+void ClimberReadyState::Run()
+{
+	m_mechanism->SetClimberMotor(0.0);
+}
+
+void ClimberReadyState::Exit()
+{
+}
+
+bool ClimberReadyState::AtTarget()
+{
+	return true;
+}
+
+bool ClimberReadyState::IsTransitionCondition(bool considerGamepadTransitions)
+{
+	// To get the current state use m_mechanism->GetCurrentState()
+
+	return (considerGamepadTransitions &&
+			!TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::CLIMBER_LOWER) &&
+			!TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::CLIMBER_RAISE));
```

Comment:
Do we need to consider motor being stalled as a transition?  I guess for now, we can skip it but we might want to think about that.

Reply:
I guess since we don't have encoder feedback on a brushed motor I don't know how we would know if the motor is stalled.